src/components/Artist.jsx

```diff
@@ -0,0 +1,14 @@
+const Artist = ({ name, externalUrl }) => {
```

Comment:
Regarding the artist names there is supposed to be a comma between the artists. You could create a function to determine whether to add a comma after the artist's name based on the index and the length of the artist array.